app/api/api/views.py

```diff
@@ -575,13 +581,22 @@ def partial_update(self, request: Any, *args: Any, **kwargs: Any) -> Response:
             else None
         )
 
-        # Send to queue
-        azure_dict = {
+        # Send to functions
+        msg = {
             "scan_report_id": scan_report_instance.id,
             "table_id": instance.id,
             "data_dictionary_blob": data_dictionary_name,
         }
-        add_message(os.environ.get("CREATE_CONCEPTS_QUEUE_NAME"), azure_dict)
+        orchestrator_url = f"{settings.AZ_FUNCTIONS_URL}/api/orchestrators/{settings.AZ_FUNCTIONS_RULES_NAME}"
+        try:
+            response = requests.post(orchestrator_url, json=msg)
+            response.raise_for_status()
+        except request.exceptions.HTTPError as e:
+            logging.error(f"HTTP Trigger failed: {e}")
+
+        # TODO: The worker_id can be used for status, but we need to save it somewhere.
+        # resp_json = response.json()
+        # worker_id = resp_json.get("id")
 
         return Response(serializer.data)
```

Comment:
Should the returned `Response` object be different and have another code depending on whether the `POST` succeeded or not?

Reply:
Short answer - ideally yes, if we fail to send to the worker, then the user should know about it. But we have nothing on the frontend to handle that anyway. 

So in our current Carrot setup no, this is the existing behaviour - you click the button, and hope it works. If it breaks, you'll never know... 
Which is related to the commented out bit - now we have a job ID that we can check for status also.